projects/storefrontapp-e2e-cypress/cypress/integration/routing-e2e-spec.ts

```diff
@@ -0,0 +1,25 @@
+const PRODUCT_ID = '1992693';
+const PRODUCT_NAME = 'DSC-T90';
+
+context('Configurable routing', () => {
+  it('should show product page by product ID', () => {
+    cy.visit(`/product/${PRODUCT_ID}`);
+    const productSummary = cy.get('cx-product-summary');
+    productSummary.should('exist');
```

Comment:
The same thing here. This exist assertion doesn't bring value.

Reply:
Removed